src/proteus/config/_compatibility.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._config import Config
+
+
+def _raise(exception: BaseException, msg: str):
+    raise exception(msg)  # type: ignore
+
+
+def _convert_spectral_file(config: Config) -> str:
+    obj = getattr(config.atmos_clim, config.atmos_clim.module)
+
+    bands = getattr(obj, 'spectral_bands')
+    group = getattr(obj, 'spectral_group')
+
+    return f'spectral_files/{group}/{bands}/{group}.sf'
+
+
+def _atmosphere_nlev(config: Config) -> str:
+    obj = getattr(config.atmos_clim, config.atmos_clim.module)
+    return obj.num_levels
+
+
+def _min_temperature(config: Config) -> str:
+    obj = getattr(config.atmos_clim, config.atmos_clim.module)
+    return obj.tmp_minimum
+
+
+def _stellar_heating(config: Config):
+    raise ValueError('`stellar_heating` has been deprecated')
+
+
+def _shallow_ocean_layer(config: Config):
+    raise ValueError("Handled by `atmos_clim.surf_state = 'mixed_layer'`")
+
+
+def _solvevol_use_params(config: Config):
+    raise ValueError("Equivalent to `delivery.initial = 'elements'`")
+
+
+def _phi_global(config: Config):
+    raise ValueError('No longer used')
+
+
+COMPAT_MAPPING = {
+    'star_model': ('star', 'mors', 'tracks'),
+    'star_mass': ('star', 'mass'),
+    'star_radius_modern': ('star', 'radius'),
+    'star_temperature_modern': ('star', 'Teff'),
+    'star_luminosity_modern': ('star', 'Lbol'),
+    'star_age_modern': ('star', 'age_now'),
+    'star_rot_pctle': ('star', 'omega'),
```

Comment:
```suggestion
    'star_rot_pctle': ('star', 'rot_pctle'),
    'star_omega': ('star', 'omega'),
```

Reply:
I'm not sure I understand this change.

In the template config there was only omega in the new config: https://github.com/FormingWorlds/PROTEUS/issues/74#issuecomment-2385333041

There is no omega in the old style config.

Do we need both omega and rot_pctle now?

Reply:
We could potentially have both. Omega usually refers to the rotation rate in something like rad/s or in omega_sun. Rotation percentile requires mors to obtain the rotation rate from the population.